classes/class-wc-connect-nux.php

```diff
@@ -186,11 +217,14 @@ public function set_up_nux_notices() {
 			}
 
 			$jetpack_install_status = $this->get_jetpack_install_status();
+			$banner_to_display = $this->get_banner_type_to_display( array(
```

Comment:
If you do decide to follow up on this route, then all my comments about renaming the constants are void :-)